ChangeLog.md

```diff
@@ -23,6 +23,10 @@ See docs/process.md for more on how version tagging works.
 - Enable use of `::` to escape port option separator (#21710)
 - In multi-threaded builds `--extern-pre-js` and `--extern-post-js` code is
   now only run on the main thread, and not on each of the workers. (#21750)
+- Multi-threaded builds no longer generate a separate `.worker.js` file, saving
+  code size and network requests.  This might require updates to build or
+  deployment scripts, since this file is no longer present in the output.
+  (#21701)
```

Comment:
What about your idea to emit a stub/almost-empty `.worker.js` file for now? I like that idea.

Reply:
Done